opam

```diff
@@ -26,7 +26,7 @@ depends: [
   "cpuid" {build}
   "ocb-stubblr" {build}
   "ounit" {test}
-  "cstruct" {>="3.0.0" & <"3.4.0"}
+  "cstruct" {>="3.0.0"}
```

Comment:
Maybe make it higher version then? Since the name had changed since that version.

Reply:
Ah, nevermind, it was introduced in 3.0.0